high_school/models.py

```diff
@@ -19,3 +19,15 @@ class HighSchool(models.Model):
 
     def __str__(self):
         return self.school_name
+
+
+class Program(models.Model):
+    high_school = models.ForeignKey(HighSchool, on_delete=models.CASCADE)
+    name = models.CharField(max_length=100, blank=False, null=False)
+    code = models.CharField(unique=True, max_length=20, blank=False, null=False)
+    description = models.CharField(max_length=2000, blank=True, null=True)
+    number_of_seats = models.IntegerField(blank=True, null=True)
+    offer_rate = models.IntegerField(blank=True, null=True)
```

Comment:
will this always be an integer or can be decimal?

Reply:
Always an integer. It's always been a whole number in the DB.